Configuration/DataProcessing/python/RecoTLR.py

```diff
@@ -60,9 +60,14 @@ def customiseDataRun2Common(process):
 def customiseDataRun2Common_25ns(process):
     process = customiseDataRun2Common(process)
 
+    import RecoLocalCalo.HcalRecAlgos.RemoveAddSevLevel as HcalRemoveAddSevLevel
+    HcalRemoveAddSevLevel.AddFlag(hcalRecAlgos,"HFDigiTime",8)
```

Comment:
the "runTheMatrix.py -l 134.802" calls the Configuration/DataProcessing/RecoTLR.customiseDataRun2Common_25ns in the main reco/pat step